distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -735,4 +738,38 @@ public void clearCatalog() {
     public void clearCache() {
         console.runCommand(CLEAR_CACHE);
     }
+
+    public ValidatableResponse executeOpenSearch(String format, String... query) {
+        StringBuilder buffer = new StringBuilder(OPENSEARCH_PATH.getUrl()).append("?")
+                .append("format=")
+                .append(format);
+
+        for (String term : query) {
+            buffer.append("&")
+                    .append(term);
+        }
+
+        String url = buffer.toString();
+        LOGGER.info("Getting response to {}", url);
+
+        return when().get(url)
+                .then();
+    }
+
+    public Response getOpenSearchAsXml(String format, String... query) {
```

Comment:
❓ Should we make this method return a `ValidatableResponse` as well? Of maybe have the other one return a simple `Response`?